lib/config/jest.js

```diff
@@ -14,7 +14,14 @@ module.exports = {
 
   rules: merge(require('./rules/jest'), {
     // Disallow vague words in test statements.
-    'shopify/jest/no-vague-titles': 'error',
+    // allowing "all"
+    'shopify/jest/no-vague-titles': [
+      'error',
+      {
+        allow: ['all'],
```

Comment:
This is basically disabled in our all our projects. Going to disable here, and eventually, we may likely just depreciate it.